Resources/Prototypes/GameRules/roundstart.yml

```
@@ -203,6 +203,27 @@
       mindRoles:
       - MindRoleTraitorReinforcement
 
+- type: entity
+  id: Changeling
+  parent: BaseGameRule
+  components:
+  - type: GameRule
+    minPlayers: 999 # TODO: make this reasonable once Changeling is out of WIP
+  - type: AntagSelection
```

```suggestion
  - type: GameRule
    minPlayers: 1 # Totally should be playable :)
  - type: AntagSelection
```

I want to play ling so bad... (Not an actual review)